app-config.yaml

```diff
@@ -75,20 +75,24 @@ scaffolder:
   # see https://backstage.io/docs/features/software-templates/configuration for software template options
 
 catalog:
-  import:
-    entityFilename: catalog-info.yaml
-    pullRequestBranchName: backstage-integration
-  rules:
-    - allow: [Component, System, API, Resource, Location]
-  locations:
-    - type: github-org
-      target: https://github.com/openedx
-      rules:
-        - allow: [User, Group]
+  # Docs here:https://backstage.io/docs/integrations/github/discovery
+  # Requires
+  # yarn add --cwd packages/backend @backstage/plugin-catalog-backend-module-github
+  providers:
+    github:
+      mainBranches:
+        organization: 'openedx'
+        filters:
+          branch: 'main'
+      masterBranches:
+        organization: 'openedx'
+        filters:
+          branch: 'master'
+      extraCatalogs:
+        organization: 'openedx'
+        catalogPath: 'catalog-extra/**/catalog-info.yaml'
```

Comment:
Does this mean the sub-catalogs live outside the code for the related component in a top-level catalog-extra folder?

Reply:
yea, that's right, this is where Dave is putting the LTI catalog file and we can add other non physical components we want to track here as well.